OpenRA.Game/Game.cs

```diff
@@ -51,7 +51,7 @@ public static OrderManager JoinServer(string host, int port, string password, bo
 		{
 			IConnection connection = new NetworkConnection(host, port);
```

Comment:
If you change this to `var connection = `... then you don't need to rely on defining `StartRecording` on the interface any more.